frontend/README.md

```diff
@@ -67,33 +65,30 @@ export const magic = createMagic();
 #### Use the Wallet module
 
 The remainder of the app uses the
-[Wallet module](https://magic.link/docs/connect/wallet-api-reference/javascript-client-sdk#wallet-module-methods)
+[Wallet module](https://magic.link/docs/api/client-side-sdks/web#wallet-module)
 `magic.wallet`. Specifically, the following three methods:
 
-1. [`connectWithUI()`](<https://magic.link/docs/connect/wallet-api-reference/javascript-client-sdk#connectwithui()>) -
-   displays the Magic Connect UI for sign up and authentication. This is called
-   when the user clicks the button to connet or sign up. You can see example
+1. [`connectWithUI()`](https://magic.link/docs/api/client-side-sdks/web#connectwithui) -
+   displays the [Magic Login UI](https://magic.link/docs/authentication/features/login-ui) for sign-up and authentication. This is called
+   when the user clicks the button to connect or sign up. You can see example
    usage in `src/components/AppHeader.tsx` and
    `src/components/LoginWithMagic.tsx`.
 
    ```jsx
    await magic.wallet.connectWithUI();
    ```
 
-2. [`showUI()`](<https://magic.link/docs/connect/wallet-api-reference/javascript-client-sdk#showui()>) -
+2. [`showUI()`](https://magic.link/docs/api/client-side-sdks/web#showui) -
    once you're connected, you can use this method to show the wallet UI. User
-   interaction is then handled by the SDK. Note that this _only_ works for Magic
-   Connect. If you support other wallets and the user connects with one of
-   those, this method will throw an error.
+   interaction is then handled by the SDK.
 
    ```jsx
    await magic.wallet.showUI().on("disconnect", () => {
      disconnect();
    });
    ```
 
-3. [`disconnect()`](<https://magic.link/docs/connect/wallet-api-reference/javascript-client-sdk#disconnect()>) -
-   disconnects from _any connected wallet_, Magic Connect or otherwise.
+3. `disconnect()` - disconnects from _any connected wallet_.
```

Comment:
`disconnect` has been deprecated. We should use [user.logout](https://magic.link/docs/api/client-side-sdks/web#logout) here instead.

Reply:
Ah I was wondering why there was no SDK method in the docs for this. Thank yoU!

Reply:
Confirmed - this `disconnect` is not the sdk method disconnect. It's handling the UI changes associated with the user having logged out. So the `.on("disconnect")` is giving a callback when teh user logs out through the widget UI so we can update accordingly.